--- lrmi.c.orig	2006-10-30 21:10:16.000000000 +0100
+++ lrmi.c	2009-02-04 19:51:44.180377549 +0100
@@ -55,6 +55,13 @@
 #include "x86-common.h"
 
 #if defined(__linux__)
+#ifndef TF_MASK
+#define TF_MASK X86_EFLAGS_TF
+#define IF_MASK X86_EFLAGS_IF
+#define IOPL_MASK X86_EFLAGS_IOPL
+#define VIF_MASK X86_EFLAGS_VIF
+#define VIP_MASK X86_EFLAGS_VIP
+#endif
 #define DEFAULT_VM86_FLAGS 	(IF_MASK | IOPL_MASK)
 #elif defined(__NetBSD__) || defined(__FreeBSD__)
 #define DEFAULT_VM86_FLAGS  (PSL_I | PSL_IOPL)
